voice/bxml/callbacks/gather.md

```diff
@@ -16,20 +16,23 @@ Content-Type: application/xml; charset=utf-8
+| applicationId    | The id of the application associated with the call.                                                                                                                                                |
+| to               | The phone number that received the call, in E.164 format (e.g. +15555555555).                                                                                                                      |
+| from             | The phone number that made the call, in E.164 format (e.g. +15555555555).                                                                                                                          |
+| direction        | The direction of the call. Either `inbound` or `outbound`. The direction of a call never changes.                                                                                                  |
+| callId           | The call id associated with the event.                                                                                                                                                             |
+| parentCallId     | (optional) If the event is related to the B leg of a `<Transfer>`, this is set to the call id of the original call leg that contained the `<Transfer>` tag. Otherwise, null.                       |
+| callUrl          | The URL of the call associated with the event.                                                                                                                                                     |
+| startTime        | Time the call was started, in ISO 8601 format.                                                                                                                                                     |
+| tag              | (optional) The `tag` specified earlier in the call. If no `tag` was specified or it was previously cleared, null.                                                                                  |
+| digits           | (optional) The digits collected from user.  Null if a timeout occurred before any digits were pressed.                                                                                             |
+| terminatingDigit | (optional) The digit the user pressed to end the gather.  Null if no terminating digit was pressed.                                                                                                |
+| transferCallerId | (optional) If the event is related to the B leg of a `<Transfer>`, this is set to the `from` field of the B-leg call, in E.164 format (e.g. +15555555555). Otherwise, null.                        |
+| transferTo       | (optional) If the event is related to the B leg of a `<Transfer>`, this is set to the phone number used as the `to` field of the B-leg call, in E.164 format (e.g. +15555555555). Otherwise, null. |
```

Comment:
```suggestion
| transferTo       | (optional) If the event is related to the B leg of a `<Transfer>`, the phone number used as the `to` field of the B-leg call in E.164 format (e.g. +15555555555). Otherwise, null.                 |
```